lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7904,187 +7904,216 @@ createUnavailableDecl(Identifier name, DeclContext *dc, Type type,
 }
 
 
+static void figureOutTheDeclarationContextToImportInto(Decl* D, ExtensionDecl *&ext, DeclContext *&DC, IterableDeclContext *&IDC);
```

Comment:
We generally do not forward declare static functions like this. But I don't have a strong opinion here.